humanguide/other-aspects/02-accessibility.rst

```diff
@@ -0,0 +1,20 @@
+Accessibility
+=============
+
+Accessibility is important to make all users to be able to use the
+operating system no matter what special needs or kind of input method
+they have.
+
+High Contrast and Colour Blind Friendly Themes
+----------------------------------------------
+
+We need to design a theme for user with special needs. One proposal of
+colour blind friendly theme is
+`here <http://colors.mivoligo.com/#color-blind>`__.
```

Comment:
"Special needs" is a term which I believe has **heavily** gone out of style. Consider, "users with color blindness".

Additionally, "We need" makes this feel a bit uncomfortable to read. Maybe "We would like help to create and implement color palettes for the following types of sight impairment: ..."

Reply:
I'm looking for a term to express persons that have less visual range, a bit more abroad. As we could address font size and contrast and not only color blindness. What is a good term here?

Reply:
Sight impairment, and "people who have sight impairment" rather than "sight-impaired people". The first option is longer but sounds better to my ear and changes the subject of the statement.

Reply:
Refrased